examples/message/SlackMessageExample.js

```diff
@@ -0,0 +1,94 @@
+const { Reshuffle } = require('reshuffle')
+const { SlackConnector, SlackEventType, SlackEvents } = require('reshuffle-slack-connector')
+const app = new Reshuffle()
+
+const main = async () => {
+  const slackConnector = new SlackConnector(app, {
```

Comment:
can you add a link to where to get these tokens from?

Reply:
added